### PR TITLE
adding opus_openoffice

### PR DESCRIPTION
**Pull request by patil-suraj**
Adding Opus OpenOffice: http://opus.nlpl.eu/OpenOffice.php
8 languages, 28 bitexts

### CODE REVIEW

File: datasets/opus_openoffice/opus_openoffice.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+from __future__ import absolute_import, division, print_function
+
+import itertools
+import os
+
+import datasets
+
+
+_CITATION = """\
+J. Tiedemann, 2012, Parallel Data, Tools and Interfaces in OPUS. \
```
Comment (lhoestq) [inline datasets/opus_openoffice/opus_openoffice.py:26]: can you use the citation in bibtex format ?

Comment (patil-suraj) [reply datasets/opus_openoffice/opus_openoffice.py]: done @lhoestq

File: datasets/opus_openoffice/README.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+- found
+languages:
+- de
+- en_GB
+- es
+- fr
+- ja
+- ru
+- sv
+- zh_CN
```
Comment (lhoestq) [inline datasets/opus_openoffice/README.md:14]: can you also format the languages depending on the config name if you don't mind ?

Sorry I didn't notice it in my previous review

**Review by lhoestq — APPROVED** [review]
awesome thanks !

**Pull request merged**